stage1/usr_from_coreos/manifest.d/systemd

```diff
@@ -1,6 +1,7 @@
 bin/coredumpctl
 bin/journalctl
 bin/systemctl
+bin/mount
```

Comment:
Is this change needed? The `.mount` files are created only in the `kvm` flavor, so it should not be needed here in the `coreos` flavor? If it is actually needed by systemd, maybe `umount` will be needed as well to stop the pod cleanly.